pandas/tests/extension/decimal/array.py

```diff
@@ -30,10 +30,13 @@ class DecimalArray(ExtensionArray):
     def __init__(self, values):
         values = np.asarray(values, dtype=object)
 
-        self.values = values
+        self._data = values
         # Some aliases for common attribute names to ensure pandas supports
         # these
-        self._items = self._data = self.data = self.values
+        self._items = self.data = self._data
```

Comment:
you have .data here? (no a big deal, but this is private yes?)

Reply:
These are just in the tests.

Reply:
What do you exactly mean? It's not really about private/public, that's up to the EA author. The purpose of this it to ensure pandas has no problems if an EA author would use `.data` to store the underlying data (in geopandas we are actually using a `data` attribute)

Reply:
I understand you *can* have a ``.data`` items, it just doesn't seem a good pattern to advertise

The fact that these are in the tests is very odd itself.

And the comment makes these doubly confusing.

Reply:
@jorisvandenbossche ok I re-read what you are testing, this is ok